docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,261 @@
+# Migrate from Pipedream v1
+
+The newest version of the Pipedream workflow introduces features like creating code steps in [Python](/code/python), [Bash](/code/bash) and [Go](/code/go).
+
+It also brings the [Node.js](/code/nodes) steps closer to our [Component API](/components/) which powers sources & actions in the platform.
+
+But this also means some of our features have changed. Here's what you need to know.
+
+## New Builder Overview
+
+Fundamentally, the new version of the workflow builder gives you the same abilities to build, test and deploy your workflows. However, there are some differences that change up your normal flow.
+
+### Editing vs Inspecting
+
+When you first open a deployed workflow, you're presented with the **inspector** version of the workflow. In this view you can see logs of past events, and select them to see the results of each step in the workflow.
+
+To edit the workflow, click the **Edit** button in the top right hand corner. This will close the inspector and allow you to edit your workflow without the distraction of logs from the production flow.
```

Comment:
Yeah re: the comment above, I guess this is the best place to say how to do it the other way: when you're in Build mode, click on the ellipsis in the top-right to get back to the Inspector

Reply:
I included a short screencast to reference the hamburger menu to switch contexts. Talking about entering, leaving and re-entering the Inspector seems a little redundant. Read through the newest version and see if that feels right.

Reply:
Makes sense and the GIF looks good